awx/main/migrations/0059_auto_20190207_2022.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-02-07 20:22
+from __future__ import unicode_literals
```

Comment:
:thinking: what is this needed for? :)

Reply:
I thought we were managing both Py 2 and 3. Removed now. Thanks!